benchmark/csharp/Tpch/TpchFunctionalQueries.cs

```diff
@@ -62,11 +65,19 @@ internal void Q1()
 
         internal void Q1v()
         {
+#if NETCOREAPP
```

Comment:
Why `#if NETCOREAPP` here?

Reply:
Since this is benchmark code, I assumed we'd want to retain the vectorized methods for core? (VectorFunctions.Intrinsics.cs exposes a vectorized `ComputeTotal` method). Since `DataFrame` is not vectorized yet, I enabled the DataFrame APIs only for framework(which was using Arrow APIs before - see changes to VectorFunctions.cs).

Reply:
Yup. Introduced `Q1d` and `Q8d` and removed the `#if NETCOREAPP`